src/numa_threadpool.rs

```diff
@@ -90,7 +90,7 @@ mod numa {
     }
 
     struct Join {}
-    type Task<Context> = Box<Fn(&Context)->() + Send + 'static>;
+    type Task<Context> = Box<dyn Fn(&Context)->() + Send + 'static>;
```

Comment:
What does this do?

Reply:
New in 2018 https://doc.rust-lang.org/edition-guide/rust-2018/new-keywords.html#dyn-trait-for-trait-objects it just makes it more explicit what will happen when you call trait objects.